### ISSUE TITLE
NewType('...', Tuple) crash

### ISSUE THREAD

**Issue by fried** [python/mypy#2613]
Code:
```
LogicalType = NewType('LogicalType', Tuple)
x = LogicalType((1,2,3))
```

Traceback: https://gist.github.com/fried/1a6ab98071d13679e266c3c73fb5af47

I would assume this is a bug because https://www.python.org/dev/peps/pep-0484/#the-any-type specifies that `Tuple` is equivalent to `Tuple[Any, ...]`.  Using `Tuple[Any, ...]` in the NewType works as expected

**Comment by ilevkivskyi** [comment]
I agree this is a bug. Earlier mypy treated bare ``Tuple`` as ``Tuple[()]``. The crash that you see is probably a remainder from the old behaviour.

**Comment by gvanrossum** [comment]
Seems to have been fixed as of 0.470 (I didn't go back and bisect to determine the exact commit).
